gtd-timer/Controllers/LogInController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+using gtdtimer.Services;
+using gtdtimer.Timer.DTO;
+
+namespace gtdtimer.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```

Comment:
change to our custom [ValidateModel] attr

Reply:
Sorry, I bad understand, what I need to change
I should add attributes Required to all field my validate model LoginDTO, yes?